src/backend/tests/change-requests.test.ts

```diff
@@ -0,0 +1,248 @@
+import request from 'supertest';
+import express from 'express';
+import prisma from '../src/prisma/prisma';
+import { batman, superman, wonderwoman } from './test-data/users.test-data';
+import changeRequestsRouter from '../src/routes/change-requests.routes';
+import {
+  changeBatmobile,
+  redesignWhip,
+  redesignWhipScopeCR,
+  redesignWhipWBSElement,
+  solutionToRedesignWhip,
+  unreviewedCr,
+  whipWorkPackage
+} from './test-data/change-requests.test-data';
+import { project1 } from './test-data/projects.test-data';
+
+const app = express();
+app.use(express.json());
+app.use('/', changeRequestsRouter);
+
+describe('Change-Requests', () => {
```

Comment:
since this whole file is for all change request tests, lets put all your tests in a nested describe that would be like `describe('reviewChangeRequest'), () => { ... } `. that once should be all one word since we're actually using the name of the endpoint function.